install_config/install/docker_registry.adoc

```diff
@@ -544,6 +558,31 @@ In the `<host>/test/busybox` example above, `test` refers to the project name.
 ====
 
 
+== Known Issues
+
+=== Image push errors with scaled registry using shared NFS volume
+
+During a build of an image you may see one of the following errors in the output of `oc build-logs BUILD`:
```

Comment:
My reasoning was that manual `docker push <internal-registry-ip>/repo/name` won't ever be as frequent as s2i build. But error suffixes are the same so I guess that making it specific to a push should be fine.